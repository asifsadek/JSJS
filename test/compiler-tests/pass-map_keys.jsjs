val a = { "foo" : "bar",
          "foo1" : "bar1" };
print(keys(a));

val b = { 1 : [1, 2, 3],
          2 : [1, 4, 9]};
print(keys(b));

val c = { true : 2,
          false : 3 };
print(keys(c));

val d = { 1 : { "foo" : 2 },
          2 : { "bar" : 3 }};
print(keys(d));
