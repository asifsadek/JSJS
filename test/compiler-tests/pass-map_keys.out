List [ "foo", "foo1" ]
List [ "1", "2" ]
List [ "true", "false" ]
List [ "1", "2" ]
